iothub/device/src/AmqpTransportSettings.cs

```diff
@@ -17,10 +17,6 @@ public sealed class AmqpTransportSettings : ITransportSettings
         private readonly TransportType _transportType;
         private TimeSpan _operationTimeout;
         private TimeSpan _openTimeout;
-        /// <summary>
-        /// To enable certificate revocation check. Default to be false.
-        /// </summary>
-        public bool CertificateRevocationCheck = false;
```

Comment:
If we shipped this bool, we can't remove it.

We could, however, proxy the value to TslVersions.Instance.CertificateRevocationCheck... treat it like a backing field.

Reply:
added changes to set it from tlsversions